activiti-api-task-runtime/src/main/java/org/activiti/api/task/runtime/TaskRuntime.java

```diff
@@ -99,7 +100,14 @@
      */
     Task complete(CompleteTaskPayload completeTaskPayload);
 
+    /**
+     * Updates the selected task with the variables set in the payload
+     * - This method checks that the task is visible by the authenticated user
+     * - This method also check that the task is assigned to the currently authenticated user
+     */
+    void updateVariables(UpdateTaskVariablesPayload updateTaskVariablesPayload);
```

Comment:
@igdianov I would call this save() and it will imply saving the task state (variables). So I would change also the name of the payloads